framework/include/markers/ReporterPointMarker.h

```diff
@@ -0,0 +1,40 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Marker.h"
+#include "ReporterInterface.h"
+
+/**
+ * Marks all elements near a given boundary for refinement/coarsening
+ */
+class ReporterPointMarker : public Marker, public ReporterInterface
+{
+public:
+  static InputParameters validParams();
+  ReporterPointMarker(const InputParameters & parameters);
+  virtual void timestepSetup() override;
+
+protected:
+  virtual MarkerValue computeElementMarker() override;
+
+  MarkerValue _inside;
+  MarkerValue _empty;
```

Comment:
```suggestion
  const MarkerValue _inside;
  const MarkerValue _empty;
```

Add docstring comment